webapp/channels/src/actions/user_actions.ts

```diff
@@ -39,8 +40,8 @@ export const queue = new PQueue({concurrency: 4});
 const dispatch = store.dispatch;
 const getState = store.getState;
 
-export function loadProfilesAndStatusesInChannel(channelId: string, page = 0, perPage: number = General.PROFILE_CHUNK_SIZE, sort = '', options = {}) {
-    return async (doDispatch: DispatchFunc) => {
+export function loadProfilesAndStatusesInChannel(channelId: string, page = 0, perPage: number = General.PROFILE_CHUNK_SIZE, sort = '', options = {}): NewActionFuncAsync { // HARRISONTODO unused
```

Comment:
Is this HARRISONTODO meant to stay?

Reply:
Just for now. They're addressed in followups. https://github.com/mattermost/mattermost/pull/26006 covers all the "unused" ones